buidl/tx.py

```diff
@@ -87,6 +87,14 @@ def dump_cache(cls, filename):
             to_dump = {k: tx.serialize().hex() for k, tx in cls.cache.items()}
             s = json.dumps(to_dump, sort_keys=True, indent=4)
             f.write(s)
+            
+    @staticmethod
+    def sendrawtransaction(signed_tx_hex_str, network="mainnet"):
+        """Broadcasts a signed transaction."""
+        url = "{}/tx".format(TxFetcher.get_url(network=network))
```

Comment:
Now that this method is moved back to `TxFetcher` this should be a `@classmethod` since you're now referencing the `cls` on this line.

Reply:
Nudge!

Reply:
Maybe I'm overlooking something?  Commit 1f01ffd changes `sendrawtransaction` to `@classmethod`.

Reply:
Sorry, what I wrote wasn't clear.

Instead of calling `TxFetcher`, this line should call `cls`:
`url = "{}/tx".format(cls.get_url(network=network))`

This is not only better for inheritance (likely irrelevant in the codebase today), but also performance as the code doesn't need to instantiate another instance of `TxFetcher`.

Reply:
Ah, yes, I missed that.  Whoops.  Addressed in commit 11f75cf.
Re-ran unit tests, and both passed.